src/org/intellij/erlang/debugger/xdebug/ErlangXDebugProcess.java

```diff
@@ -358,6 +361,25 @@ private OSProcessHandler runDebugTarget() throws ExecutionException {
     return erlangProcessHandler;
   }
 
+  private void setupErlangAppConfig(GeneralCommandLine commandLine) throws ExecutionException {
+    ErlangRunConfigurationBase<?> runConfig = getRunConfiguration();
+    if (runConfig.getDebugOptions().isLoadingConfig()) {
+      commandLine.addParameters("-config", runConfig.getDebugOptions().getAppConfig());
+    }
+  }
+
+  private void setupErlangRebarDependencies(GeneralCommandLine commandLine) throws ExecutionException {
+    ErlangRunConfigurationBase<?> runConfig = getRunConfiguration();
+    if (runConfig.getDebugOptions().isIncludingRebarDependencies()) {
+      Set<String> dependencies = RebarConfigUtil.getRebarDependencies(
+              runConfig.getConfigurationModule().getModule(),
+              runConfig.getDebugOptions().isFetchingDependencies());
+
+      for (String dep : dependencies)
+        commandLine.addParameters("-pa", dep);
+    }
```

Comment:
> and second - re-importing project to update rebar dependencies.

I ran into alot of issues including dependencies upon import. For some odd reason, when I run the import, the plugin ends up nesting dependencies, recursively calling rebar get-deps for each dependency. The directory structure for deps ends up looking like this:

deps/amqp_client/deps/rabbit_common

Instead of:

deps/amqp_client
deps/rabbit_common

This layout prevented some paths from being included at runtime as only dependencies at the top level directory structure in deps were included, "-pa".

I don't know if that was intentional or not, but it wasn't working for our project setups. I don;t know, maybe i can test it more, but pretty certain this was the case.

I have no problems separating the config option into run configs. However, reimporting projects definitely doesn't seem to setup the path correctly for running eunit tests and applications, which require deps. The rebar run configuration did run with deps correctly, but there is no debug feature for the rebar run configuration.